schemas/database/database.md

```diff
@@ -0,0 +1,36 @@
+# Databases client spans
+
+Calls to databases should be tracked as client spans.
+
+Span `name` should be set to low cardinality value representing the statement
+executed on database. It may be stored procedure name (without argument), sql
+statement without variable arguments, etc. When it's impossible to get any
+meaningful representation of the span `name`, it can be populated using the same
+value as `db.instance`.
+
+**TODO: we might need have separate specs for other database types like Redis
+and such.**
+
+Note, Redis, Cassandra, HBase and other storage systems may reuse the same
+attribute names.
+
+**TODO: Agree to use `type` instead of `component`?**
+
+| Attribute name | Notes and examples |
+|:---------------|:-------------------|
+| `type`         | Database driver name or database name (when known) `JDBI`, `jdbc`, `odbc`, `postgreSQL`. |
+| `db.type`      | Database type. For any SQL database, `"sql"`. For others, the lower-case database category, e.g. `"cassandra"`, `"hbase"`, or `"redis"`. |
```

Comment:
Wondering if `nosql` could also be recommended ;)

Reply:
just general case `nosql`? I think you would alweays know more than just it's not a `sql` =)